sinker/sinker.go

```diff
@@ -117,6 +117,61 @@ func (s *MongoSinker) HandleBlockUndoSignal(ctx context.Context, data *pbsubstre
 	return fmt.Errorf("received undo signal but there is no handling of undo, this is because you used `--undo-buffer-size=0` which is invalid right now")
 }
 
+func (s *MongoSinker) ParseFieldsIntoDatabaseTypes(fields []*pbdatabase.Field, table string) (parsedFields map[string]interface{}, err error) {
+  parsedFields = make(map[string]interface{})
```

Comment:
Seems we might have some alignment issue here. Also, let pre-allocate the `map`:
```suggestion
    parsedFields = make(map[string]interface{}, len(fields))
```